lona/html/attribute_dict.py

```diff
@@ -71,13 +71,16 @@ def __getitem__(self, name):
 
     def __setitem__(self, name, value, issuer=None):
         if not isinstance(value, (int, bool, float, str)):
-            raise ValueError('unsupported type: {}'.format(type(value)))
+            raise ValueError(f'unsupported type: {type(value)}')
 
-        if name in ('id', 'class', 'style'):
```

Comment:
Great change, but this is a functional change. I would like a dedicated commit with commit message for it (just append it to this pr)

Reply:
Fixed